dags/taar_weekly.py

```diff
@@ -30,6 +46,128 @@
+        """/google-cloud-sdk/bin/gsutil -m rm gs://%s/* || /bin/true"""
+        % TAAR_ETL_STORAGE_BUCKET,
+    ]
+
+
+wipe_gcs_bucket = GKEPodOperator(
+    owner="vng@mozilla.com",
+    email=["vng@mozilla.com", "mlopatka@mozilla.com",],
+    task_id="wipe_taar_gcs_bucket",
+    name="wipe_taar_gcs_bucket",
+    image="google/cloud-sdk:242.0.0-alpine",
+    arguments=wipe_gcs_files(),
+    location="us-central1-a",
+    cluster_name="bq-load-gke-1",
+    dag=taar_weekly,
+)
+
+dump_bq_to_tmp_table = GKEPodOperator(
+    owner="vng@mozilla.com",
+    email=["vng@mozilla.com", "mlopatka@mozilla.com",],
+    task_id="dump_bq_to_tmp_table",
+    name="dump_bq_to_tmp_table",
+    # This uses a circleci built docker image from github.com/mozilla/taar_gcp_etl
+    image=TAAR_ETL_CONTAINER_IMAGE,
+    arguments=[
+        "-m",
+        "taar_etl.taar_profile_bigtable",
+        "--iso-date={{ ds_nodash }}",
+        "--gcp-project=%s" % TAAR_PROFILE_PROJECT_ID,
+        "--avro-gcs-bucket=%s" % TAAR_ETL_STORAGE_BUCKET,
+        "--bigtable-instance-id=%s" % TAAR_BIGTABLE_INSTANCE_ID,
+        "--sample-rate=1.0",
+        "--fill-bq",
+    ],
+    dag=taar_weekly,
+)
+
+extract_bq_tmp_to_gcs_avro = GKEPodOperator(
+    owner="vng@mozilla.com",
+    email=["vng@mozilla.com", "mlopatka@mozilla.com",],
+    task_id="extract_bq_tmp_to_gcs_avro",
+    name="extract_bq_tmp_to_gcs_avro",
+    # This uses a circleci built docker image from github.com/mozilla/taar_gcp_etl
+    image=TAAR_ETL_CONTAINER_IMAGE,
+    arguments=[
+        "-m",
+        "taar_etl.taar_profile_bigtable",
+        "--iso-date={{ ds_nodash }}",
+        "--gcp-project=%s" % TAAR_PROFILE_PROJECT_ID,
+        "--avro-gcs-bucket=%s" % TAAR_ETL_STORAGE_BUCKET,
+        "--bigtable-instance-id=%s" % TAAR_BIGTABLE_INSTANCE_ID,
+        "--bq-to-gcs",
+    ],
+    dag=taar_weekly,
+)
+
+dataflow_import_avro_to_bigtable = GKEPodOperator(
+    owner="vng@mozilla.com",
+    email=["vng@mozilla.com", "mlopatka@mozilla.com",],
+    task_id="dataflow_import_avro_to_bigtable",
+    name="dataflow_import_avro_to_bigtable",
+    # This uses a circleci built docker image from github.com/mozilla/taar_gcp_etl
+    image=TAAR_ETL_CONTAINER_IMAGE,
+    # Due to the nature of the container run, we set get_logs to False,
+    # To avoid urllib3.exceptions.ProtocolError: 'Connection broken: IncompleteRead(0 bytes read)' errors
+    # Where the pod continues to run, but airflow loses its connection and sets the status to Failed
+    # See: https://github.com/mozilla/telemetry-airflow/issues/844
+    get_logs=False,
+    arguments=[
+        "-m",
+        "taar_etl.taar_profile_bigtable",
+        "--iso-date={{ ds_nodash }}",
+        "--gcp-project=%s" % TAAR_PROFILE_PROJECT_ID,
+        "--avro-gcs-bucket=%s" % TAAR_ETL_STORAGE_BUCKET,
+        "--bigtable-instance-id=%s" % TAAR_BIGTABLE_INSTANCE_ID,
+        "--subnetwork=regions/us-west1/subnetworks/gke-taar-nonprod-v1",
```

Comment:
Use variable `taar_dataflow_subnetwork` instead.

Reply:
`taar_dataflow_subnetwork` is an airflow variable.

Reply:
Fixed in 97111e9b5